linux_os/guide/services/ssh/ssh_server/sshd_use_approved_kex_ordered_stig/ansible/shared.yml

```diff
@@ -0,0 +1,18 @@
+# platform = Red Hat Enterprise Linux 7,Oracle Linux 7
```

Comment:
You are adding this rule to RHEL 8 and OL 8 profiles, but you haven't created remediation scripts for RHEL 8 and OL 8.  Do you plan to extend the Ansible and Bash to work on these systems?

Reply:
I was not planning to add remediations for OL8 and RHEL8 since it would require to modify the files in `/etc/crypto-policies/back-ends`

Maybe adding a warning message like the one in `configure_openssl_tls_crypto_policy` makes sense:
```
This rule doesn't come with a remediation, automatically changing the crypto-policies may be too disruptive [...]
```

Reply:
I added the warning